app/Console/Commands/Sync.php

```diff
@@ -83,7 +84,7 @@ public function handle(): int
 					$deleteImported,
 					$this->option('import_via_symlink') === '1',
 					$importViaSymlink,
-					$this->option('resync_metadata')
+					(bool) $this->option('resync_metadata')
```

Comment:
Here we have another example for these inconsistencies. Two lines above we have `$this->option(...) === '1'`, here we have `(bool) $this->option(...)`.